### PR TITLE
Initial suggested settings for pagination

### PR DESCRIPTION
**Pull request by andreasfrisch**
closes dfds/cloudplatform/issues/2109

# Additional Review Notes
 The values for each table's default size are pure guess work.

image
image

### CODE REVIEW

File: src/src/pages/capabilities/MyCapabilities.js [hunk 1]
```diff
@@ -210,11 +213,21 @@ export default function MyCapabilities() {
...
+                }
+                }}
+              muiBottomToolbarProps={{
+                sx: {
+                  background: 'none',
+                }
+              }}
+              enableDensityToggle={false}
+              enableHiding={false}
+              enableFilters={true}
```
Comment (HrisyToteva) [inline src/src/pages/capabilities/MyCapabilities.js:229]: image

The pop up filters are not aligned (it is not nice for the eye). The last one (arrow) should not be shown.

Comment (andreasfrisch) [reply src/src/pages/capabilities/MyCapabilities.js:235]: Cleaned up as per our real life conversation

**Comment by andreasfrisch** [comment]
image

Search bars like this instead of top bar options

**Comment by HrisyToteva** [comment]
I've noticed one thing, and it's more of a question. In our other capabilities and on the topic page, we highlight the text during a search, but we don't do this in the 'My Capabilities' table. Should we enable this feature everywhere and be consistent?

**Pull request merged**